### PR TITLE
Strengthen Python() tests

### PR DESCRIPTION
**Pull request by convoliution**
## Impact and Context

Previously, a handful of the tests in `test_versioning/test_environment.py` only checked whether the `environment`/`constraints` field of `Python`'s underlying protobuf was simply populated. This PR robustifies those tests to verify expected contents.

The `Python.requirements` and `Python.constraints` properties have been updated/added to return a user's logged items, and facilitate testing.

## Risks

None are known at present.

## Testing

- [ ] ~Deployed the service to dev env~
  - this change is client-only
- [ ] ~Used functionality on dev env~ 
  - this change is client-only
- [x] Added unit test(s)
  - `pytest test_versioning/test_environment.py` passes in Python 2 and 3 on my machine
- [ ] ~Added integration test(s)~
  - this change is client-only

### CODE REVIEW

**Comment by convoliution** [comment]
Try again Jenkins

**Comment by convoliution** [comment]
try again jenkins

**Comment by convoliution** [comment]
Try again Jenkins

**Comment by conradoverta** [comment]
!rebuild

**Comment by convoliution** [comment]
try again jenkins

**Comment by convoliution** [comment]
try again  jenkins

File: client/verta/verta/environment/_python.py [hunk 1]
```diff
@@ -132,13 +112,33 @@ def __repr__(self):
...
+            # parsed requirements
+            return sorted(
+                map(
+                    self._req_spec_msg_to_str,
+                    self._msg.python.requirements,
+                )
+            )
+        else:
+            # raw requirements
+            return self._msg.python.raw_requirements.splitlines()
```
Comment (conradoverta) [inline client/verta/verta/environment/_python.py:141]: See here one example where we assume 1 line == 1 requirement.

**Review by conradoverta — APPROVED** [review]
LGTM

**Pull request merged**